katas/main/src/pokerhands/jorge/Players.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace PokerGame.Hands.Jorge;
+
+public class PokerGameBlackAndWhite
+{
+    public string BothHands(string competitorsHand)
+    {
+        double blackHand = new PokerHand().HandToPlay(competitorsHand.Substring(7, 14));
+        double whiteHand = new PokerHand().HandToPlay(competitorsHand.Substring(30, 14));
+
+        return blackHand > whiteHand
+            ? "Black wins - with " + resultsScore[blackHand]
+            : "White wins - with " + resultsScore[whiteHand];
+    }
+
+    public Dictionary<double, string> resultsScore = new Dictionary<double, string>
```

Comment:
I would recommend turn result score into a enum, it helps to prevent magic numbers